Jenkinsfile

```diff
@@ -43,11 +43,17 @@ pipeline {
         archiveArtifacts artifacts: "${REPO}/**/pdf/*.pdf", fingerprint: true, allowEmptyArchive: true
       }
     }
-    // stage('Tests') {
-    //   steps {
-    //     runXmostest("${REPO}", 'tests')
-    //   }
-    // }
+    stage('Tests') {
+      steps {
+        viewEnv() {
+          withVenv{
+            dir("${REPO}/tests") {
```

Comment:
It looks like dir() needs to be before withVenv because it searches up through directories for a .venv directory to activate the virtual environment - but in this case the current directory hasn't been set so it starts at the root of the sandbox, and the .venv directory will be lib_xud so searching upwards won't find it.

I would expect it to work if you re-order them like this:
```
    dir("${REPO}/tests") {
      viewEnv() {
        withVenv {
          ...
```

Reply:
Thanks.